schema/deploy/shipping/views.sql

```diff
@@ -225,4 +225,33 @@ create or replace view shipping.observation_with_presence_absence_result_v1 as
       join shipping.presence_absence_result_v1 using (sample)
       order by site, encounter, sample, target;
 
+
+create or replace view shipping.flu_assembly_jobs_v1 as
+
+    select sample.identifier as sfs_uuid,
+           sample.details ->> 'nwgc_id' as nwgc_id,
+           sequence_read_set.urls,
+           target.identifier as target,
+           o1.lineage as target_linked_organism,
+           case (o2.lineage) is not null
+             when true
+             then o2.lineage
+             else o1.lineage
+           end as assembly_job_organism,
+           sequence_read_set.details
+
+      from warehouse.sequence_read_set
+      join warehouse.sample using (sample_id)
+      join warehouse.presence_absence using (sample_id)
+      join warehouse.target using (target_id)
+      join warehouse.organism o1 using (organism_id)
+      left join warehouse.organism o2 on (o2.lineage ~ concat(o1.lineage::text, '.*{1}')::lquery)
+
+    where target.identifier like 'Flu%'
```

Comment:
I think it'd be more robust to filter on the target's lineage since that will be more stable than the target identifier, which may not always start with `Flu` in the future.